### PR TITLE
feature: external links [NP-1382]

### PR DESCRIPTION
**Pull request by baseonmars**
Screenshot 2020-12-10 at 14 26 44

### CODE REVIEW

File: styleguide/foundations/typography/typography.stories.mdx [hunk 1]
```diff
@@ -170,6 +170,30 @@ export const linkTextHtml = `<p class="my-paragraph"><a href="">cads-hyperlink</
 
 Read our guidance about using <a href ="?path=/docs/components-buttons--primary">buttons as links</a>.
 
+### External Links
+
+When you link to a page external to your website add the `.cads-hyperlink--external` class to display an icon after the link. This helps to signals to a user that they are leaving the current website.
```
Comment (baseonmars) [inline styleguide/foundations/typography/typography.stories.mdx:175]: This reason seems to be disputed internally.

File: scss/5-objects/_prose.scss [hunk 2]
```diff
@@ -51,4 +51,9 @@
   .cads-list-no-bullet {
     @extend %cads-list-no-bullet;
   }
+
+  a[href*="//"]:not([href*="citizensadvice.org.uk"])::after
```
Comment (davidrapson) [inline scss/5-objects/_prose.scss:55]: Gnarly, but I guess this makes sense

**Review by davidrapson — APPROVED** [review]
👍  I imagine the rationale for the icon will need updating as you mentioned. I think it's more "to maintain parity with existing pages" as these icons are ambiguous. No doubt @soniaturcotte or @minniessuyuhuang could frame that better.

**Comment by baseonmars** [comment]
> 👍 I imagine the rationale for the icon will need updating as you mentioned. I think it's more "to maintain parity with existing pages" as these icons are generally speaking ambiguous. No doubt @soniaturcotte or @minniessuyuhuang could frame that better.

I'll remove it. We should be able to stand behind our decisions rather than give a weak reason.

**Pull request merged**